data/json/monsters/zed-animal.json

```diff
@@ -548,7 +548,7 @@
     "type": "MONSTER",
     "name": { "str": "Zeindeer" },
     "looks_like": "mon_zeer",
-    "description": " A once majestic Caribou, this undead creature barely resembles the reindeer of tradition and seems to only be interested in human flesh.",
+    "description": " A once-majestic caribou, this undead creature barely resembles the reindeer of tradition and seems to only be interested in human flesh.",
```

Comment:
Is the initial space here intentional, or typo?

Reply:
It was like that when I first looked at the file; it's not a change I introduced, so I can't answer that. I missed it in my initial pass.